src/Altinn.Platform/Altinn.Platform.Storage/DataCleanup/LoadTestingCleanup.cs

```diff
@@ -48,15 +50,19 @@ public LoadTestingCleanup(ICosmosService cosmosService, IBlobService blobService
                 return new BadRequestObjectResult("Pass an app name in the query string to clean up load testing data.");
             }
 
+            Stopwatch stopwatch = new Stopwatch();
+            stopwatch.Start();
             List<Instance> instances = await _cosmosService.GetAllInstancesOfApp(app.ToLower().Trim());
 
+            int successfullyDeleted = 0;
+
             foreach (Instance instance in instances)
             {
                 bool dataElementsDeleted, dataElementMetadataDeleted = false;
 
                 try
                 {
-                    if (instance.Data.Count > 0)
+                    if (instance.Data?.Any() == true)
```

Comment:
We can't use the Data property to indicate if there are data elements on the instance. Data elements are stored separately. Two possible directions:
1. Update GetAllInstnaceesOfApp to always update the Data property with correct list of data elements. 
2. Assume that all instances have data elements and undoncitionally call DeleteDataBlob 
 and DeleteDataElementDocuments 

The DeleteDataBlob and DeleteDataElementDocuments don't require the DataElement document and are smart enough not to fail if an instance don't have any. Any exception will prevent the DeleteInstanceDocument from being called.